BioMed.Api/BioMed.Infrastructure/Persistence/Configuration/VisitEntityConfiguration.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BioMed.Infrastructure.Persistence.Configuration
+{
+    internal class VisitEntityConfiguration
+    {
+    }
+}
```

Comment:
Why are configurations not written for **VisitEntityConfiguration**?

Reply:
I will write now